ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/jboss/as/ee/main/module.xml

```diff
@@ -15,6 +15,9 @@
     <!-- /exports -->
 
     <resources>
+        <!-- TODO preview module will instead pack org.wildfly:wildfly-concurrency-impl-31 -->
+        <artifact name="${org.wildfly:wildfly-concurrency-impl-30}"/>
```

Comment:
This artifact should be provided by its own module. That module should be what differs between an EE 10 installation and an EE 11 one. That avoids duplicating this complex module.xml

Modules can have circular dependencies so that should work.

Reply:
this will be done in another commit, since I am dealing with some issues when I break up this module